src/pages/verify/guides/privatekey-jwt.mdx

```diff
@@ -0,0 +1,160 @@
+---
+product: verify
+category: Guides & Tools
+sort: 1
+title: Private key JWT authentication
+subtitle: How to use private key JWT to authenticate your Criipto application
+---
+
+import Layout from '../../../layouts/mdx';
+
+export default Layout;
+
+Private Key JWT Authentication is one of the client authentication methods defined in [Open ID Connect Core Client Authentication](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication).
+It relies on using JWT assertions signed with asymmetric key pairs for client authentication.
+
+<Highlight icon="file-lines">
+
+Private Key JWT Authentication (`private_key_jwt`) offers a [more secure alternative](/verify/guides/privatekey-jwt/#advantages-of-private-key-jwt-authentication) to symmetric credentials (`client_secret` authentication options)
+for client authentication in [traditional server-based web applications](/verify/getting-started/oidc-intro/#using-openid-connect-to-integrate-with-criipto-verify).
+
+</Highlight>
+
+You can configure your [Criitp Verify Application](https://docs.criipto.com/verify/getting-started/basics/#applications) to use `private_key_jwt` for client authentication.
+
+## Authenticate with Private Key JWT
+
+To use `private_key_jwt` for client authentication, follow these three steps:
+
+1. Generate an RSA key pair and a JSON Web Key Set(JWKS).
```

Comment:
Maybe these three steps are a bit superfluous, as they are each addressed in the subsequent sub-headings?